instrumentation/github.com/aws/aws-sdk-go/service/otels3/mocks/S3API.go

```diff
@@ -0,0 +1,7440 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Is this truly necessary?  This is a very large inclusion for something that looks like it could be achieved with the same mock implementation in the example package.